tf2_sensor_msgs/test/test_tf2_sensor_msgs.py

```diff
@@ -29,61 +29,106 @@
 # POSSIBILITY OF SUCH DAMAGE.
 
 import copy
-import struct
 import unittest
 
-from sensor_msgs import point_cloud2
-from sensor_msgs.msg import PointField
+import numpy as np
+from sensor_msgs_py.point_cloud2 import create_cloud_xyz32, read_points
+from std_msgs.msg import Header
 from tf2_ros import TransformStamped
-import tf2_sensor_msgs
+from tf2_sensor_msgs.tf2_sensor_msgs import do_transform_cloud, transform_points
```

Comment:
If the public API is imported into the `tf2_sensor_msgs/__init__.py` scope, you can do this:
```suggestion
from tf2_sensor_msgs import do_transform_cloud, transform_points
```